src/DMAPI/tgs/v5/bridge.dm

```diff
@@ -107,3 +107,45 @@
 		return
 
 	return bridge_response
+
+#define TGS_BRIDGE_WRAP_RETURNED 1
+#define TGS_BRIDGE_WRAP_RESPONSE 2
+
+/// world.Export() can hang forever, so we need to give a timeout window.
+/datum/tgs_api/v5/proc/ActuallyPerformBridgeRequest(bridge_request)
+	var/list/bridge_request_tracker = list(
+		TGS_BRIDGE_WRAP_RETURNED = FALSE,
+		TGS_BRIDGE_WRAP_RESPONSE = null
+	)
+
+	var/start_time = world.time
+	while(bridge_request_tracker[TGS_BRIDGE_WRAP_RETURNED] == FALSE)
```

Comment:
```suggestion
	while(!bridge_request_tracker[TGS_BRIDGE_WRAP_RETURNED])
```

Reply:
I deliberately compare for 1/0 so that entries not present (null) are differentiated. This is deliberate.